bundles/mapping/drawtools/plugin/DrawPlugin.ol3.js

```diff
@@ -327,11 +339,23 @@ Oskari.clazz.define(
                    me.sendDrawingEvent(me._id, optionsForDrawingEvent);
                    return geometry;
                  }
+            } else if (shape === 'point' || shape === 'Point') {

```

Comment:
This should be done to all or none of the types. And all the places that reference the shapes. I think we should only use 'Point' as with the other shapes.